src/detect-http-server-body.c

```diff
@@ -39,7 +39,11 @@
 #include "detect-parse.h"
 #include "detect-engine.h"
 #include "detect-engine-mpm.h"
+#include "detect-engine-mpm.h"
```

Comment:
Most of these added includes aren't necessary.

Reply:
Can you do a follow up PR?

Reply:
Will do.